rosdep/python.yaml

```diff
@@ -8647,6 +8647,15 @@ python3-venv:
     pip:
       packages: []
   ubuntu: [python3-venv]
+python3.8-venv:
```

Comment:
See https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules

Reply:
we intentionally add python3.8-venv, because on melodic(20.04), default python3 uses version 3.6, but sometimes we would like to install python3.8 for development purpose.

Reply:
That sounds reasonable to me, though it's the first time we're adding a rule like this.
Let me check with other people of the team if it is fine.

I'm also not sure about the name of the key, though the proposed one seems fine.

Reply:
I don't really think we should go down this road.  As you can see from the automated feedback, that package doesn't exist in Ubuntu Bionic, nor does it exist on any other operating system.  In order for that to work, you'd need to add separate keys for `python3.8` itself (and a new place to get it), plus all of the pip packages that you wanted to go along with that.  It doesn't seem like a scalable solution, so I really don't think we should do this.